example_sqrtlasso.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+from numpy.linalg import norm
+import matplotlib.pyplot as plt
+from statsmodels.regression import linear_model
+
+from skglm.penalties import L1
+from skglm.datafits.single_task import SqrtQuadratic
```

Comment:
This line is currently failing for me, I guess this should be replaced by `from skglm.experimental.sqrt_lasso import SqrtLasso`.
Overall it seems that file is redundant with `_plot_sqrt_lasso.py` and could be deleted

Reply:
I'll delete it indeed